### DISCUSSION TITLE
chia_harvester fault

### DISCUSSION THREAD

**Discussion by asd2317419** [Chia-Network/chia-blockchain#1728] [Q&A]
![QQ截图20210405180623](https://user-images.githubusercontent.com/81959665/113786599-07ea7480-976c-11eb-9b66-88d8a96335e7.png)
![QQ截图20210407064042](https://user-images.githubusercontent.com/81959665/113786732-56980e80-976c-11eb-8747-3c74206cd746.png)
![QQ截图20210407064133](https://user-images.githubusercontent.com/81959665/113786741-5a2b9580-976c-11eb-9b80-9f77b8362ab8.png)

**Comment by wjblanke** [comment]
what version of chia are you running?
